sass/_components.scss

```diff
@@ -0,0 +1,174 @@
+.about {
+  &__container {
+    padding: 0 1.6rem 6rem 1.6rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    position: relative;
+    top: 4rem;
+
+    @media screen and (min-width: 768px) {
+      top: 10rem;
+      max-width: 80rem;
+      margin: 0 auto;
+      justify-content: flex-start;
+    }
+  }
+
+  .hero__welcome {
+    background: #fff;
+    border: 1px solid #e6e6e6;
+  }
+
+  &__contact {
+    font-size: 1.6rem;
+    margin-top: 2rem;
+    font-weight: 400;
+    color: var(--color-secondary-dark);
+  }
+
+  &__email {
+    text-decoration: underline;
+    color: var(--color-primary);
+    font-weight: 700;
+    font-size: 1.4rem;
+  }
+}
+
+.logo {
+  display: flex;
+  justify-content: center;
+  background-color: #fff;
+  flex-direction: column;
+  text-align: center;
+  padding: 4rem 1.6rem;
+  align-items: center;
+
+  h3 {
+    font-size: 2.2rem;
+    font-weight: bolder;
+  }
+
+  hr {
+    width: 60px;
+    border: solid 1px var(--color-tertiary);
+    margin: 1rem auto;
+  }
+
+  p {
+    font-size: 1.6rem;
+    font-weight: 400;
+    margin-top: 1rem;
+    margin-bottom: 2rem;
+  }
+
+  img {
+    width: 100px;
+    height: auto;
+    margin-top: 1rem;
+    border: 1px solid var(--color-tertiary);
+    box-sizing: content-box;
+    padding: 1.6rem 7rem;
+  }
+}
+
+.history {
+  background-color: #fff;
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```

Comment:
- [ ] **Optional:** As mentioned above the rule `align-items: center;` could also be extracted into its class or made into one of your variables.